core/classes/DbQuery.class.php

```diff
@@ -952,6 +952,7 @@ public function sql_like( $p_alias, $p_pattern, $p_escape = null, $p_force_ci =
 					$t_operator = 'ILIKE';
 					break;
 				case DB_TYPE_ORACLE:
+				case DB_TYPE_SQLITE3:
```

Comment:
Update above comment

Reply:
To be precise for SQLite we need to use external functions for non-ASCII characters... 🤔
I'd rather not make this change, but add a comment and TODO to use [mb_strtoupper()](https://www.php.net/manual/en/function.mb-strtoupper.php).

Reply:
> use [mb_strtoupper()](https://www.php.net/manual/en/function.mb-strtoupper.php).

Makes sense

Reply:
I checked and found that the case-insensitive variant of sql_like() is only used in sql_ilike(), which is only used to look up e-mails. So basic SQLite functionality is enoguh.